PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -3052,14 +3064,11 @@ int Progress::thread()
 
 int Progress::createProgressWindow()
 {
-	DWORD styleEx = WS_EX_OVERLAPPEDWINDOW;
-	if (_hOwner)
-		styleEx |= WS_EX_TOOLWINDOW;
-
-	_hwnd = ::CreateWindowEx(styleEx,
+	_hwnd = ::CreateWindowEx(
+		WS_EX_TOOLWINDOW | WS_EX_OVERLAPPEDWINDOW | WS_EX_TOPMOST,
 		cClassName, _header, WS_POPUP | WS_CAPTION,
 		CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT,
-		_hOwner, NULL, _hInst, (LPVOID)this);
+		NULL, NULL, _hInst, (LPVOID)this);

```

Comment:
Can you drop the `(LPVOID)` cast? It [shouldn't be necessary](http://eli.thegreenplace.net/2009/11/16/void-and-casts-in-c-and-c).